doc/content/api/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "API"
+description: ""
+aliases: ["/api/reference/grpc"]
+weight: 7
+menu:
+  main:
+    weight: 7
+---
+
+{{% tts %}} is API first by design and has rich support for various kinds of APIs. This section introduces the reader to the different types of APIs supported by {{% tts %}}, basic concepts required to use the APIs and full reference list of all the supported APIs.
+
+<!-- more -->
+
+<div class="fixed-table table-api">
+
+| Type                                                      | Description                                                                                                              |
+| --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| [HTTP (REST)]({{< ref "/api/reference/http" >}})          | {{% tts %}} HTTP (REST) APIs are primarily used for managing entities (ex: Devices, gateways, users, organizations etc). |
+| [gRPC]({{< ref "/api/reference/grpc" >}})                 | {{% tts %}} backend components communicate with each other primarily using [gRPC](https://grpc.io/).                     |
+| [LoRa® Gateway]({{< ref "/api/concepts/lora-gateway" >}}) | APIs for LoRa® Gateways to communicate with {{% tts %}}.                                                                 |
+| [Data Plane]({{< ref "/integrations" >}})                 | APIs to read/receive application data and send downlinks to devices.                                                     |
```

Comment:
Just call this Application Integrations or Integrations. We don't use the term Data Plane for that here.